### PR TITLE
Remove warnings

### PR DESCRIPTION
**Pull request by dev7ch**
- remove $lodash warings
- set default status to draft on creation if draft exists
- exclude more data in .gitignore

### CODE REVIEW

File: src/interfaces/status/input.vue [hunk 1]
```diff
@@ -59,6 +59,12 @@ export default {
     }
   },
   created() {
+    if (!this.value || this.value === "") {
+      // Set  "Draft" to default if the option exists
+      if (this.$store.state.permissions[this.collection].statuses.draft !== undefined) {
+        this.$emit("input", "draft");
```
Comment (rijkvanzanten) [inline src/interfaces/status/input.vue]: We shouldn't hardcode this here for just the name `draft`. The default can theoretically be set to anything

File: src/routes/file-library.vue [hunk 2]
```diff
@@ -278,6 +278,9 @@ export default {
     }
   },
   methods: {
+    keyBy(items, selector) {
```
Comment (rijkvanzanten) [inline src/routes/file-library.vue]: I think we should be able to access it by adding it in like

```js
methods: {
  keyBy: _.keyBy
}
```

**Comment by rijkvanzanten** [comment]
Thanks @dev7ch! The default for status might be a bit troubling though..

File: src/interfaces/status/input.vue (hunk 1 above)
Comment (dev7ch) [reply src/interfaces/status/input.vue]: yes, its true, but actually i was unsure ttoo about this.

But is there already way so set default status ?

like simply `default: true` ??

File: src/routes/file-library.vue (hunk 2 above)
Comment (dev7ch) [reply src/routes/file-library.vue]: yes should work too

File: src/interfaces/status/input.vue (hunk 1 above)
Comment (dev7ch) [reply src/interfaces/status/input.vue]: and i think if this not exist there shouldn`t an error, its like "pseudo hardcoded"  i would say;)

Comment (rijkvanzanten) [reply src/interfaces/status/input.vue]: You can set the default for the status interface by using the `default value` field when setting up the interface. We can default the status to the _first_ of the statuses, but we shouldn't do it based on the name of the status, as that's user generated content.

Comment (dev7ch) [reply src/interfaces/status/input.vue]: Ok i see

**Pull request merged**